set.mm

```diff
@@ -17334,12 +17341,6 @@ arbitrary numbers of universal quantifiers in the consequent (the series
   wel $p wff x e. y $=
     ( cv wcel ) ACBCD $.
 
-  $( Register class-to-set promotion and class equality and membership as
-     primitive expressions. Although these are actually definitions, the above
-     ambiguity prevention necessitates our taking class equality as the
-     primitive, instead of set equality. $)
-  $( $j primitive 'weq' 'wel'; $)
-
```

Comment:
It looks like we're losing some important information here:
> Although these are actually definitions, the above
     ambiguity prevention necessitates our taking class equality as the
     primitive, instead of set equality.

Reply:
Yes, the comment was very much out of date, note that the things in the "Register..." sentence do not match the `primitive` line. It's basically just recapitulating the comment immediately above it.

Reply:
Indeed, the primitive is the set equality `weq` and not the class equality `wceq`, so this comment was wrong, thanks!